editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/flag/FlagListFragment.java

```diff
@@ -0,0 +1,116 @@
+package ca.ubc.eml.soiltopargraphy.editor.ui.flag;
+
+import android.arch.lifecycle.ViewModelProviders;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.util.DiffUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.support.v7.recyclerview.extensions.ListAdapter;
+
+import ca.ubc.eml.soiltopargraphy.editor.R;
+
+/**
+ * Fragment where the list of flags (and their image and descriptions) are displayed
+ */
+
+public class FlagListFragment extends Fragment {
+
+    public static FlagListFragment newInstance() {
+        return new FlagListFragment();
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.flag_list_fragment, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        FlagListViewModel flagViewModel = ViewModelProviders.of(this).get(FlagListViewModel.class);
+        FlagAdapter flagAdapter = new FlagAdapter();
+        flagAdapter.submitList(flagViewModel.getFlagList());
+    }
+
+}
+
+
+class FlagViewHolder extends RecyclerView.ViewHolder {
+    public TextView name;
```

Comment:
Now that these properties are only used in FlagViewHolder, we can make them private.